conanfile.py

```
@@ -17,22 +24,30 @@ def requirements(self):
     def build_requirements(self):
         self.tool_requires("cmake/[>=3.27.0]")
 
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
     def layout(self):
         cmake_layout(self)
 
+    def generate(self):
+        deps = CMakeDeps(self)
```

Ruff (0.8.2)

Undefined name `CMakeDeps`

(F821)